.github/workflows/upload_plugin.yml

```diff
@@ -48,14 +48,14 @@ jobs:
         if: ${{ env.DRY_RUN != 'false' }}
         uses: goreleaser/goreleaser-action@v6
         with:
-          version: 1.26.2
+          version: latest
           args: build --snapshot
       - name: Build binaries
         # Only one way to not dry run - see 'false'. All other cases, conservatively assume --dry-run
         if: ${{ env.DRY_RUN == 'false' }}
         uses: goreleaser/goreleaser-action@v6
         with:
-          version: 1.26.2
+          version: latest
```

Comment:
nit: add a note to the PR description (or commit message) on why it's fine to use `latest` here, to save readers from having to go to https://goreleaser.com/blog/goreleaser-v2/#github-action to find out

Reply:
i've removed `version:` so default is used which is `latest`: 3275e72